pkg/controller/ratelimiter/ratelimiter.go

```diff
@@ -69,3 +71,16 @@ func RequeueRateLimiter() ratelimiter.RateLimiter {
 		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(5), 50)},
 	)
 }
+
+// SetMasterRateLimiter sets the the kubernetes client level rate limiter.
```

Comment:
Nit: Probably something like `SetOverallRateLimiter` to avoid the "master" term, but not a blocker